ISHPermissionKit/Requests/ISHPermissionRequestUserNotification.m

```diff
@@ -65,6 +59,7 @@ - (ISHPermissionState)permissionStateForNotificationSettings:(UNNotificationSett
         case UNAuthorizationStatusDenied:
             return ISHPermissionStateDenied;
 
+        case UNAuthorizationStatusProvisional:
```

Comment:
This is important:

You can ask for provisional permission (you must do this explicitly via `desiredOptions`, which will allow you to send one quite (no alert/sound, but in the notification center) notification. Users can change their settings directly from notification center.

After requesting provisional permission, the auth state stays in `UNAuthorizationStatusProvisional` until the user has made a decision. During that time, you may also request general permission, which is why we treat it as "NotDetermined".

Reply:
thanks for the detailed explanation. This makes for a great comment in the code I reckon. Should be valuable later on when we come across this and 🤔.